src/row.js

```diff
@@ -75,24 +91,37 @@ export class Row {
     this.isRanked === 'NO';
 
   isRowClean = () =>
+    this.teamPlayerName1 &&
+    this.teamPlayerName2 &&
```

Comment:
Could ensure here if row is clean by checking teamPlayerName3 if bracked is 3s (and same logic for 5s and p4 p5)?

Reply:
Could be nice, but 2 problems: 
- I'm supporting multi-brackets so I can do that only if 3s is the only selected bracket. If 2s and 3s are selected, then teamPlayerName3 could, or could not be there. 
- currently we clean data before applying season/bracket filters (but we could apply some additional post-filter cleaning)
But there are still advanced cleanup cases we could handle, pre-filters: 
- if we have teamPlayerName4 but not 5, there is a problem (no 4s bracket) - and same for enemyPlayerClass
- if we have some team name but no matching enemy class, there is a problem too